src/ssh_audit/ssh_audit.py

```diff
@@ -836,82 +839,129 @@ def process_commandline(out: OutputBuffer, args: List[str], usage_cb: Callable[.
-        usage_cb(out, str(err))
     aconf.ssh1, aconf.ssh2 = False, False
     host: str = ''
     oport: Optional[str] = None
-    port: int = 0
-    for o, a in opts:
-        if o in ('-h', '--help'):
+
+    parser = argparse.ArgumentParser(description='SSH Audit Tool', add_help=False)
+    
+    # Add short options to the parser
+    parser.add_argument('-1', '--ssh1', action="store_true", dest='ssh1', default=None)
+    parser.add_argument('-2', '--ssh2', action="store_true", dest='ssh2', default=None)
+    parser.add_argument('-4', '--ipv4', action="store_true", dest='ipv4', default=None)
+    parser.add_argument('-6', '--ipv6', action="store_true", dest='ipv6', default=None)
+    parser.add_argument('-b', '--batch', action="store_true", dest='batch', default=None)
+    parser.add_argument('-c', '--client-audit', action="store_true", dest='client_audit', default=None)
+    parser.add_argument('-d', '--debug', action="store_true", dest='debug', default=None)
+    parser.add_argument('-g', '--gex-test', action="store", dest='gex_test', default=None)
+    parser.add_argument('-h', '--help', action="store_true", dest='help', default=None)
+    parser.add_argument('-ip', '--ip-address', '--hostname', action="store", dest='host', type=str)
+    parser.add_argument('-j', '--json', action="store_true", dest='json', default=None)
+    parser.add_argument('-jj', '--json-indent', action="store_true", dest='json_indent', default=None)
+    parser.add_argument('-l', '--level', action="store", dest='level', type=str, default='info')
+    parser.add_argument('-L', '--list-policies', action="store_true", dest='list_policies', default=None)
+    parser.add_argument('-M', '--make-policy', action="store", dest='make_policy', default=None)
+    parser.add_argument('-m', '--manual', action="store_true", dest='manual', default=None)
+    parser.add_argument('-P', '--policy', action="store", dest='policy', default=None)
+    parser.add_argument('-p', '--port', action="store", dest='port', default='22', type=int)
+    parser.add_argument('-T', '--targets', action="store", dest='targets', default=None)
+    parser.add_argument('-t', '--timeout', action="store", dest='timeout', default='5', type=int)
+    parser.add_argument('-v', '--verbose', action="store_true", dest='verbose', default=None)
+
+
+    # Add long options to the parser
+    parser.add_argument('--conn-rate-test', action="store", dest='conn_rate_test', default='0', type=int)
+    parser.add_argument('--dheat', action="store", dest='dheat', default='0', type=int)
+    parser.add_argument('--lookup', action="store", dest='lookup', default=None)
+    parser.add_argument('--no-colors', action="store_true", dest='no_colors', default=None)
+    parser.add_argument('--skip-rate-test', action="store_true", dest='skip_rate_test', default=None)
+    parser.add_argument('--threads', action="store", dest='threads', default='1', type=int)
+
+    try:
+        namespace = parser.parse_args()
```

Comment:
Calling this variable `args` might improve readability.

Reply:
Updating code